05_book_titles/book.rb

```diff
@@ -1,3 +1,16 @@
+#create class called BOOK
 class Book
-# write your code here
-end
+   attr_reader :title
+    
+  def title=string
+    articles = %w[a an and if or in the of]
+    words = string.split(" ")
+  
+    words.map do |word|
```

Comment:
Similar to your code for 02_calculator, `map` does more than you need.

```suggestion
    words.each do |word|
```